doc/node.1

```diff
@@ -281,6 +277,10 @@ with small\-icu support.
 When set to \fI1\fR, process warnings are silenced.
 
 .TP
+.BR NODE_NO_HTTP2 =\fI1\fR
+When set to \fI1\fR, the http2 module is suppressed.
```

Comment:
s/supressed/not included in the runtime?

Not 100% that "supressed" gets the message across

Reply:
Suggestion: `the 'http2' module is not exposed by the runtime`
If decided to change should mirror in doc/api/cli.md:440 and src/node.cc:3832